### DISCUSSION TITLE
Welcome to maplibre-gl-directions Discussions!

### DISCUSSION THREAD

**Discussion by smellyshovel** [maplibre/maplibre-gl-directions#73] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by clementmas** [comment]
I'll start. I'm the solo founder of TravelMap and I use this great package to help users trace and share their travel itineraries: https://travelmap.net/itinerary

I'm a full-stack web dev but I especially like working with maps (MapLibre GL JS, Leaflet, etc.).
